Cargo.toml

```diff
@@ -36,7 +36,7 @@ walkdir = "^2.2"
 num_cpus = "^1.8"
 semver = "^0.9"
 md5 = "^0.6"
-zip = { version = "^0.4", features = ["deflate-zlib"], default-features = false }
+zip = "^0.5"
```

Comment:
I used the features thing so it would reduce the size of the resulting binary.

Reply:
If the features didn't change with 0.5, it'd be nice to keep that.

Reply:
I tried but it doesn't seem that it is possible to do that anymore?!

Reply:
Ah it looks like the feature name changed from `deflate-zlib` to `deflate`.

Reply:
As far as I remember, there were some performance issues with the lastest zip lib.
So we need to be sure that we don't have a too big perf regression (if one of course).